app/models/katello/concerns/host_managed_extensions.rb

```diff
@@ -542,9 +542,9 @@ def rhel_eos_schedule_index
 
       def package_names_for_job_template(action:, search:, versions: nil)
         if self.operatingsystem.family == 'Debian'
-          deb_names_for_job_template(action, search)
+          deb_names_for_job_template(action: action, search: search)
         else
-          yum_names_for_job_template(action, search, versions)
+          yum_names_for_job_template(action: action, search: search, versions: versions)
```

Comment:
why not

Reply:
Oh this seems to come from this PR https://github.com/Katello/katello/pull/10845